releasenotes/notes/0.10/0-10-release-8c37dadcc1c82fcc.yaml

```diff
@@ -1,66 +1,66 @@
 ---
-prelude: >
+prelude: |
```

Comment:
Does this actually work? `prelude: >` is the default created by reno when you do `reno new <>`

Reply:
`>` joins everything onto one line, `|` reads it verbatim.  To get bullet-point lists to work, you need `|`.